src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -127,6 +127,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if ($this->defaultViewsPath) {
             $viewsPaths[] = $this->defaultViewsPath;
         }
+        $controllersPaths = array($kernel->getRootDir().'/Controller');
```

Comment:
What if my controllers are somewhere else?

Reply:
@iltar you're right. I added a parameter to constructor (as for the views). But I'm not sure how to test it. Can you help me?

Reply:
What about making the parameter an array of paths? That would cover multiple locations. You can test it by writing fixtures for your test, writing controllers with translations.

Reply:
An array of paths sounds great to me, but for the tests I wonder how to set path(s) when I call the command on CLI (bin/console translation:update)?

When I try your suggestion, it returns null for $this->router.
I'm not sure it's OK to have a behaviour for templates (parameter in constructor) and another behaviour for controllers (find auto).